osquery/tables/system/darwin/disk_encryption.cpp

```diff
@@ -26,6 +28,62 @@ const std::string kDeviceNamePrefix = "/dev/";
 // kCoreStorageIsEncryptedKey is not publicly defined
 // or documented because CoreStorage is a private framework
 #define kCoreStorageIsEncryptedKey_ "CoreStorage Encrypted"
+#define kIODeviceTreeChosenPath_ "IODeviceTree:/chosen"
+
+Status genUnlockIdent(CFDataRef& uuid) {
```

Comment:
Curious if this could be `const`? I see it's passed to other CF methods so perhaps not.